elvis.config

```diff
@@ -4,7 +4,7 @@
       [ #{ dirs    => ["src"]
          , filter  => "*.erl"
          , ruleset => erl_files
-         , rules   => [ {elvis_style, god_modules, #{ignore => [els_client]}}
+         , rules   => [ {elvis_style, god_modules, #{ignore => [els_client,els_methods]}}
```

Comment:
There's no space after the comma. Should we run elvis on the elvis.config file? :D